components/ws-manager/pkg/manager/create.go

```diff
@@ -881,6 +881,7 @@ func (m *Manager) newStartWorkspaceContext(ctx context.Context, req *api.StartWo
 
 	workspaceSpan := opentracing.StartSpan("workspace", opentracing.FollowsFrom(opentracing.SpanFromContext(ctx).Context()))
 	traceID := tracing.GetTraceID(workspaceSpan)
+	defer tracing.FinishSpan(workspaceSpan, &err)
```

Comment:
This not great - the intent of this span is to live over the course of the workspace lifecycle until the workspace hits `running`.